cloudevents/service.go

```diff
@@ -17,16 +16,19 @@ import (
 	"time"
 
 	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/rs/zerolog/log"
 )
 
 const (
+	DefaultLogLevel       = LogDebug
```

Comment:
isn't INFO a more standard default value? logging events by default means that everyone want to set it to something else for prod

Reply:
same as `http`

Reply:
I agree that `INFO` is more sensible, moving forward - so I'd than change that outside of _this_ PR

Reply:
Ok, I thought it was new

Reply:
Yes @pierDipi , Info will be the default once we have the environment variable plumbed through from the middleware to enable debug logging on-demand.  For the short term, it has to be set as the default to debug.  👍🏻